.env.test

```diff
@@ -1,6 +1,8 @@
 DEBUG=true
 SECRET_KEY='tests'
 DATABASE_URL='postgres://localhost/opencraft'
+INSTANCE_MYSQL_URL='mysql://root@localhost'
+INSTANCE_MONGO_URL='mongodb://localhost'
```

Comment:
I assumed that the hostname can be resolved from both the instance manager and the instances. If that is not the case I can add another setting. `localhost` here is just for testing.